src/swagger/subscribe/get.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @swagger
+ * /subscribers:
+ *   get:
+ *     tags:
+ *       - Subscribers
+ *     summary: Retrieves a list of all subscribers.
+ *     description: This endpoint retrieves all the subscribers from the database.
+ *     responses:
+ *       200:
+ *         description: A list of subscribers.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 status:
+ *                   type: string
+ *                   example: success
+ *                 message:
+ *                   type: string
+ *                   example: Subscribers found successfully
+ *                 result:
+ *                   type: array
+ *                   items:
+ *                     $ref: '#/components/schemas/Subscriber'
```

Comment:
result is shown as "string" which is incorrect. It should show the same type as the return type

Reply:
but the type is an array which is the same as the return type

Reply:
with components defined above, the problem should be solved.